components/layouts/TheMenu/index.vue

```diff
@@ -51,33 +51,34 @@
             :class="{ 'menu__logo_active': isActiveHome(homeLink) }"
           />
         </AppLink>
-        <AppLink
-          v-for="link in mainLevelLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-          kind="secondary"
-        >
-          {{ link.label }}
-        </AppLink>
-      </nav>
-    </section>
-    <section
-      v-if="isCommunityActive()"
-      class="menu__second-level"
-    >
-      <nav class="menu__navigation-level">
-        <AppLink
-          v-for="link in communitySubLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-          kind="secondary"
-        >
-          {{ link.label }}
-        </AppLink>
+        <ul class="menu__navigation-level__list">
+          <li v-for="link in mainLevelLinks" :key="link.url" class="menu__navigation-level__list-item">
+            <BasicLink
+              v-if="!link.sublinks"
+              class="menu__link"
+              :class="{ 'menu__link_active': isActive(link) }"
+              v-bind="link"
+            >
+              {{ link.label }}
+            </BasicLink>
+            <cv-dropdown v-else class="cv-dropdown menu__navigation__dropdown" :class="{ 'menu__link_active': isCommunityActive() }" placeholder="Community">
```

Comment:
You don't have CSS for `cv-dropdown` nor `menu__navigation__dropdown` so I think you don't need them